### PR TITLE
Rules Result-and-Readme

### PR DESCRIPTION
**Pull request by MayankKumar10**
Adding Rules Result-and-Readme into Vision Quiz

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me!

File: src/components/Result.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+				class="padding-small-card shade classA translateOne"
+			>
+      <div
+				class="padding-small-card shade flex-column-al-center translateOne margin-normal-bottom back-btn"
+		>score: 6/10 </div>
+					<div
+						class="quiz-container flex-row-center padding-small-card shade"
+					>
+
+          <img class="quiz-result-img imgTransition" src="../components/img/trophy3.jpg" alt="">
```
Comment (partha8) [inline src/components/Result.html]: you have a missing alt here

**Review by partha8 — APPROVED** [review]

File: src/components/Result.html (hunk 1 above)
Comment (MayankKumar10) [reply src/components/Result.html]: I'll do that. Thank you, Partha for taking some spare Time and giving PR review.

**Comment by MayankKumar10** [comment]
> Looks good to me!

Thank you, Apurva for taking some spare Time and giving PR review.

**Pull request merged**